internal/testshared/testshared.go

```diff
@@ -392,13 +392,22 @@ func testConsumeWithoutAck(t *testing.T, ts TestServer) {
 }
 
 func testProduceStatusOk(t *testing.T, ts TestServer) {
-	t.Skip("missmatch grpc version")
 	assert := assert.New(t)
 
 	// very basic happy path test.
 	prod := ts.NewProducer(generateID())
-	status, err := prod.Status()
-	assert.NoError(err)
```

Comment:
Do we know the exact change triggering this failure?

Reply:
The change in grpc library version caused this, it seems to report as `connecting` in the test rather than connected, so it needs to wait a little while now before asserting.